lib/terraforming/resource/db_security_group.rb

```diff
@@ -46,7 +46,7 @@ def ingresses_of(security_group)
       end
 
       def db_security_groups
-        @client.describe_db_security_groups.db_security_groups.select { |sg| ingresses_of(sg).length > 0 }
+        @client.describe_db_security_groups.map(&:db_security_groups).flatten.select { |sg| ingresses_of(sg).length > 0 }
```

Comment:
__[rubocop]__
* Line is too long. [118/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Use `!empty?` instead of `length > 0`.